application/ui/src/features/inspect/dataset/dataset-status-panel.component.tsx

```diff
@@ -81,78 +78,100 @@ const ReadyToTrain = ({ onStartTraining, isPending }: ReadyToTrainProps) => {
 };
 
 interface TrainingInProgressProps {
-    jobId: string;
+    job: Job;
 }
 
-const TrainingInProgress = ({ jobId }: TrainingInProgressProps) => {
+const REFETCH_INTERVAL_WITH_TRAINING = 1_000;
+
+const useProjectTrainingJobs = () => {
+    const { projectId } = useProjectIdentifier();
+
     const { data } = $api.useQuery('get', '/api/jobs', undefined, {
         refetchInterval: ({ state }) => {
-            const job = state.data?.jobs.find(({ id }) => id === jobId);
+            const job = state.data?.jobs.find(
```

Comment:
minor: you could use `some` and then call the variable `projectHasTrainingJob` or something similar. (my immediate reaction to seeing `.find !== undefined` is that it could be replaced by `some`)